frontend/src/components/GddSend/TransactionForm.vue

```diff
@@ -11,7 +11,13 @@
                 </b-form-radio>
               </b-col>
               <b-col>
-                <b-form-radio v-model="selected" name="radios" :value="sendTypes.link" size="lg">
+                <b-form-radio
+                  v-model="selected"
+                  name="radios"
+                  :value="sendTypes.link"
+                  size="lg"
+                  @change="form.email = ''"
```

Comment:
Better avoid validation of email field. Easiest way should be to use `v-if` instead of `v-show`

Reply:
change v-show to v-if. 
deleted @change  
this also solves the problem with the non-valid email